core/sql/regress/charsets/TEST014

```diff
@@ -28,6 +28,11 @@
 --
 -- @@@ END COPYRIGHT @@@
 
+-- an existing bug causes process loop when infer_charset is ON
```

Comment:
Is there a JIRA for this bug? Do we know anything about the bug (other than this test runs into it)?

Reply:
jira TRAFODION-2730 tracks this issue.